dns/dnsperfgo/main.go

```diff
@@ -223,6 +235,12 @@ func (c *dnsClient) runQuery(name string, timeout time.Duration, lookupFunc Look
 		startTime := time.Now()
 		_, err := lookupFunc(name)
 		latency := time.Since(startTime)
+		if infoLogger != nil && err == nil {
+			infoLogger.Info("DNS lookup successful, latency recorded",
```

Comment:
suggest:

infoLogger("lookup", "name", name, "latency_ms", latency.Milliseconds(), "err", err)

Latency we want in milliseconds
Record the error.